### ISSUE TITLE
Error installing remix-auth-github

### ISSUE THREAD

**Issue by saurabhp75** [sergiodxa/remix-auth-github#33]
I am getting error when installing remix-auth-github.

```
$ npm i remix-auth-github
npm ERR! ERESOLVE could not resolve
npm ERR! 
.
.
.
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! remix-auth-github@"*" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @remix-run/server-runtime@1.19.3
npm ERR! node_modules/@remix-run/server-runtime
npm ERR!   peer @remix-run/server-runtime@"^1.0.0" from remix-auth-github@1.6.0
npm ERR!   node_modules/remix-auth-github
npm ERR!     remix-auth-github@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by bluefire2121** [comment]
The peer dependencies need to be updated. Maybe this can coincide with the change to vite.

**Comment by rgildea** [comment]
Also having this issue. Is anyone using this package?

**Comment by sergiodxa** [comment]
This should be fixed in [v1.7.0](https://github.com/sergiodxa/remix-auth-github/releases/tag/v1.7.0).
